src/xmipp/libraries/parallel/mpi_classify_CLTomo_prog.cpp

```diff
@@ -703,7 +703,7 @@ void CL3D::write(const FileName &fnRoot, int level) const
             row.setValue(MDL_ANGLE_PSI, assignment.psi);
             SFq.addRow(row);
         }
-        MetaData SFq_sorted;
+        MetaDataDb SFq_sorted;
         SFq_sorted.sort(SFq, MDL_IMAGE);
```

Comment:
`sort` is implemented only for the `Db` version?

Reply:
It is implemented for `vec` version too (and is part of common API). I don't know why i kept db implementation here. Changed to vec implementation, code compiles.